tests/test_viewers.py

```diff
@@ -105,6 +105,8 @@ def test_structure_data_viewer_storage(structure_data_object):
         -1.6859999895095825, -1.6859999895095825, -0.6669999957084656, 1,
     ]
     # fmt: on
+    # Avoid producing temporary files from povray in the repo
+    monkeypatch.chdir(tmp_path)
```

Comment:
This test produces `Si2.png` and `__temp__.pov` files so we change the current working directory to a tmp_path. Ultimately this should probably be fixed somewhere in the `_render_structure` code.